### PR TITLE
Clean gitignore

### PR DESCRIPTION
**Pull request by froboy**
Two tiny tweaks:
- `.idea` had two entries, so removing the second.
- Drupal's `.gz` files seemed to be _un-ignored_ which seems... not good? I ran `composer install` and I got a bunch of those files to commit to the repo. That seems wrong, right?

## Affected URL
None

## Related Tickets
None

## Steps to Validate
1. Open project in PHPStorm, observe no `.idea` files are shown by git.
1. Run `composer install`. Observe no new files are shown by git.

## Deploy Notes
NA

**New scripts**:

- `script` : script details

**New dependencies**:

- `dependency` : dependency details

**New dev dependencies**:

- `dependency` : dependency details

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -101,11 +101,5 @@ Thumbs.db
 ##########
 .sass-cache
 
-# Things in the core directory that Drupal 8 commits in the repository.
-!web/core/**/*.gz
-
-# Ignore files generated by PhpStorm
-/.idea/
```
Comment (sean-e-dietrich) [inline .gitignore]: @froboy why do we remove php storm?

Comment (sean-e-dietrich) [reply .gitignore]: also wondering if we should remove the `.vscode` directory

Comment (froboy) [reply .gitignore]: Not removing altogether, just removing a dupe.
https://github.com/kanopi/starter-8/blob/15e076fe21821144e6d530042d6eea9b3da6f15c/.gitignore#L67

Comment (sean-e-dietrich) [reply .gitignore]: Ahh ok!

**Pull request merged**